src/components/schema/jsonld.tsx

```diff
@@ -0,0 +1,92 @@
+import {LocalBusiness, Thing, WithContext, PostalAddress} from 'schema-dts';
+import PhotoGallery from '../photo-gallery';
+
+export function JsonLd<T extends Thing>(json: WithContext<T>): string {
+  return `<script type="application/ld+json">
+${JSON.stringify(json)}
+</script>`;
+}
+
+export const SchemaWrapper = (data: any) => {
+  return LocalBusinessSchemaWrapper(data)
+}
+
+export const LocalBusinessSchemaWrapper = (data: any) => {
+  return JsonLd<LocalBusiness>({
+    '@context': 'https://schema.org',
+    '@type': 'LocalBusiness', // TODO: change this to match the current project's business type
+    name: data.document.streamOutput.name,
+    address: parseAddress(data.document.streamOutput.address),
+    description: data.document.streamOutput.description,
+    telephone: data.document.streamOutput.mainPhone,
+    image: parsePhotoGallery(data.document.streamOutput.photoGallery),
+    openingHours: parseOpeningHours(data.document.streamOutput.hours),
+    paymentAccepted: data.document.streamOutput.paymentOptions,
+    makesOffer: data.document.streamOutput.services,
+  });  
+}
+
+// takes in a list of yext images and return a list of image urls
+export const parsePhotoGallery = (gallery: any) => {
+  let imageArray = new Array<string>();
+
+  for (const photo of gallery) {
+    imageArray.push(photo.image.url)
+  }
+
+  return imageArray;
+}
+
+export const parseAddress = (address: any): PostalAddress => {
```

Comment:
Can you `TODO` these types, we can replace any with Address once we define it in the component repo